phpBB/phpbb/search/fulltext_sphinx.php

```diff
@@ -331,11 +331,16 @@ protected function config_generate()
 				array('docinfo',					'extern'),
 				array('morphology',					'none'),
 				array('stopwords',					''),
+				array('wordforms',					'  # optional, specify path to wordforms file. See ./docs/sphinx_wordforms.txt for example'),
```

Comment:
Should these comments be inside the strings? 
Wouldn't it be more logical to have them behind them?
```
array('wordforms', ''), // optional, specify path to wordforms file. See ./docs/sphinx_wordforms.txt for example
```

Reply:
They are inside strings because the sample configuration file is generated within a window in the ACP (for the user to copy and paste into a configuration file) and so the comments need to appear there (rather than within fulltext_sphinx.php).

Reply:
Oh okay, that makes perfect sense! Cheers.